docs/reference/avalanchego/info-api.md

```diff
@@ -22,6 +22,97 @@ This API uses the `json 2.0` RPC format. For more information on making JSON RPC
 
 ## Methods
 
+### `info.acps`
+
+Returns peer votes for Avalanche Change Proposals (ACPs).
+
+**Signature:**
+
+```sh
+info.acps() -> {
+    acps: map[uint32]{
+        supportWeight: uint64
+        supporters:    map[ids.NodeID]
```

Comment:
nb: this is supposed to be a set datatype but i'm not sure how to represent this since i'm not sure what format this is.